forge-gui/res/cardsfolder/upcoming/space_marine_scout.txt

```diff
@@ -0,0 +1,11 @@
+Name:Space Marine Scout
+ManaCost:2 W
+Types:Creature Astartes Scout
+PT:2/1
+K:First Strike
+K:Vigilance
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChange | TriggerDescription$ Concealed Position — When CARDNAME enters the battlefield, if an opponent controls more lands than you, you may search your library for a Plains card, put it onto the battlefield tapped, then shuffle.
+SVar:TrigChange:DB$ ChangeZone | CheckSVar$ X | SVarCompare$ GTY | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Plains | ChangeNum$ 1 | Tapped$ True
```

Comment:
CheckSVar logic + placement wrong

Might want to look at other cards with this condition, they all do it correctly